elements/pfe-button/demo/index.html

```diff
@@ -0,0 +1,181 @@
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.3.0/webcomponents-bundle.js"></script>
+
+    <!-- Tests require pulling in the UMD version of the files -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.5/require.min.js"></script>
+
+    <script>require([
+      "../dist/pfe-button.umd.js",
+      "../../pfe-band/dist/pfe-band.umd.min.js"
+    ])</script>
+
+    <style>
+      input {
+        font-size: 16px;
+        border: 0;
+        border-radius: 3px;
+        padding: 8px;
+      }
+
+      #disablers-container {
+        display: flex;
+      }
+
+      #disablers-container > div {
+        display: flex;
+        flex-direction: column;
+        padding-right: 16px;
+      }
+    </style>
+  </head>
+  <body unresolved>
+    <pfe-band pfe-color="lightest">
+      <header slot="pfe-band--header">
+        <h2>&lt;pfe-button&gt;</h2>
+      </header>
+      <article>
+        <pfe-button>
+          <button disabled>Primary</button>
+        </pfe-button>
+        <pfe-button pfe-variant="secondary">
+          <button>Secondary</button>
+        </pfe-button>
+        <pfe-button pfe-variant="tertiary">
+          <button>Tertiary</button>
+        </pfe-button>
+        <pfe-button pfe-variant="danger">
+          <button>Danger</button>
+        </pfe-button>
+      </article>
+    </pfe-band>
+
+    <pfe-band>
+      <header slot="pfe-band--header">
+        <h2>&lt;pfe-button&gt; in a form</h2>
+      </header>
+      <article>
+        <form id="form">
+          <input type="text" placeholder="name" />
+          <pfe-button>
+            <button>Submit</button>
+          </pfe-button>
+        </form>
+        <div id="formSubmittedMessage" hidden>The form was submitted</div>
+      </article>
+    </pfe-band>
+
+    <pfe-band pfe-color="lightest">
+      <header slot="pfe-band--header">
+        <h2>&lt;pfe-button&gt; Disabled</h2>
+      </header>
+      <article id="disablers-container">
+        <div>
+          <pfe-button>
+            <button>Primary</button>
+          </pfe-button>
+          <label>
+            <input type="checkbox"> Disable
+          </label>
+        </div>
+        <div>
+          <pfe-button pfe-variant="secondary">
+            <button>Secondary</button>
+          </pfe-button>
+          <label>
+            <input type="checkbox"> Disable
+          </label>
+        </div>
+        <div>
+          <pfe-button pfe-variant="tertiary">
+            <button>Tertiary</button>
+          </pfe-button>
+          <label>
+            <input type="checkbox"> Disable
+          </label>
+        </div>
+        <div>
+          <pfe-button pfe-variant="danger">
+            <button>Danger</button>
+          </pfe-button>
+          <label>
+            <input type="checkbox"> Disable
+          </label>
+        </div>
+      </article>
+    </pfe-band>
+
+    <pfe-band>
```

Comment:
@kylebuch8 When you get a chance, can you include test where the light DOM button has different types: https://www.w3schools.com/tags/tag_button.asp#midcontentadcontainer